src/Azure.Functions.Cli/Actions/AzureActions/BaseAzureAction.cs

```diff
@@ -125,6 +126,12 @@ private async Task<(bool, string)> TryGetAzCLIManagementURL()
 
         private async Task<string> GetAccessToken()
         {
+            if (EnvironmentHelper.GetEnvironmentVariableAsBool(Constants.E2ETestConstants.EnableDeploymentTests))
```

Comment:
Can this move below the `TryGetAzCliToken` and `TryGetAzPowerShellToken`, so you're sure you're not altering existing behavior, then come up with some descriptive env var name, like `AZURE_MANAGEMENT_ACCESS_TOKEN` and we can just use that as another source of the access token.